app/components/wds-avatar-stack.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+import {computed} from '@ember/object';
+
+
+export default Component.extend({
+	classNames: ['wds-avatar-stack'],
```

Comment:
avatars could be a property on the compent as a null
so it still blows when you don't pass it
but this way when you look at the component you see what properties it have

Reply:
done